### PR TITLE
Tripwire Belt, a MAIDS Replacement

### PR DESCRIPTION
**Pull request by ValoTheValo**
## About The Pull Request
This PR replaces the M.A.I.D.S meme background with something more serious and in line with the rest of the backgrounds. i also edited First fleet slightly because Humon told me too
20-Jun-22 14-06-37

## Why It's Good For The Game
The maids background was an overpowered meme and was not in line with the other backgrounds. it gave more points than any other and was not a location. that plus
 the meme name and premise meant it didn't fit in Eris. Tripwire Belt is a background that is in line with the others in the game, not giving more than 5 extra points in total and is a location with a serious premise.
 
20-Jun-22 13-55-15

## Changelog
:cl:
add: Added Tripwire Belt Background
del: Removed MAIDS Background
balance: slightly edits First Fleet Background
/:cl:

### CODE REVIEW

**Comment by ValoTheValo** [comment]
Image of the background is one commit behind the last grammar fixes. everything is capitalized fine now in game

**Review by SirRichardFrancis — COMMENTED** [review]
Commit overdose.
Diagnosis: Github Desktop user.

File: code/datums/setup_option/backgrounds/origin.dm [hunk 1]
```diff
@@ -240,18 +240,15 @@
...
-			At other times, they're assigned to Ironhammer security teams on exploratory missions into the null sector due to their prowess as construction specialists and medics."
+/datum/category_item/setup_option/background/origin/tripwire
+	name = "Tripwire Belt"
+	desc = "A network of hidden ships, gun emplacements and listening bases on a large asteroid belt late in the Corporate War by the Syndicate for the purpose of being an early warning station for attacks on Hanza Sector core worlds. After the end of the corporate war IronHammer transformed it into a large system of training camps and bases for specialists and new recruits. Due to an almost complete lack of terraforming, hostile environment and outdated syndicate construction, the Tripwire Belt is suited for teaching new recruits how to survive in deepspace conditions, but this also means morale is often low."
```
Comment (Firefox13) [inline code/datums/setup_option/backgrounds/origin.dm:245]:
```suggestion
	desc = "A network of hidden ships, gun emplacements and listening bases placed in a large asteroid belt late in the Corporate War by the Syndicate for the purpose of being an early warning station for attacks on Hansa Sector core worlds. After the end of the corporate war Ironhammer transformed it into a large system of training camps and bases for specialists and new recruits. Due to an almost complete lack of terraforming, hostile environment and outdated syndicate construction, the Tripwire Belt is suited for teaching new recruits on how to survive in deep space conditions, but this also means morale is often low."
```

**Review by Firefox13 — COMMENTED** [review]
also this:
изображение

**Comment by ValoTheValo** [comment]
> also this: изображение

did i fuck up my math? dammit

**Comment by Firefox13** [comment]
> did i fuck up my math? dammit

Yes, they're both at 0 total now

**Comment by ValoTheValo** [comment]
> > did i fuck up my math? dammit
> 
> Yes, they're both at 0 total now

amazing. gonna try and fix that.

**Comment by ValoTheValo** [comment]
fixed

**Review by TheShown911 — COMMENTED** [review]
Please clean up your commits.

**Comment by ValoTheValo** [comment]
> Please clean up your commits.

Is there a way to do it without setting up an entirely new repo and using a different client?

**Comment by SirRichardFrancis** [comment]
> Is there a way to do it without setting up an entirely new repo

Yes.

> and using a different client?

No.

**Pull request merged**